/**:
  ros__parameters:
    update_rate: 10
    min_acceleration: -1.0
    max_jerk: 0.3
    min_jerk: -0.3
